plugins/lighthouse/src/components/AuditList/AuditListTable.tsx

```diff
@@ -13,81 +13,107 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React, { FC, useMemo } from 'react';
-import {
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TableRow,
-} from '@material-ui/core';
-import { makeStyles } from '@material-ui/core/styles';
-import { Website } from '../../api';
+import React, { FC, useState } from 'react';
+import { Table, TableColumn, TrendLine, useApi } from '@backstage/core';
+import { Website, lighthouseApiRef } from '../../api';
+import { useInterval } from 'react-use';
 import {
+  formatTime,
   CATEGORIES,
   CATEGORY_LABELS,
-  SparklinesDataByCategory,
   buildSparklinesDataForItem,
 } from '../../utils';
-import Audit from '../Audit';
-
-const useStyles = makeStyles(theme => ({
-  table: {
-    minWidth: 650,
-  },
-  status: {
-    textTransform: 'capitalize',
-  },
-  link: {
-    paddingTop: theme.spacing(2),
-    paddingBottom: theme.spacing(2),
-    display: 'inline-block',
-  },
-  statusCell: { whiteSpace: 'nowrap' },
-  sparklinesCell: { minWidth: 120 },
-}));
+import { Link } from '@material-ui/core';
+import AuditStatusIcon from '../AuditStatusIcon';
 
 export const AuditListTable: FC<{ items: Website[] }> = ({ items }) => {
```

Comment:
Was a bit disappoint that the new version of this file was longer, until I realized the Audit and AuditRow are gone :grin: